pyanaconda/modules/payloads/constants.py

```diff
@@ -31,6 +31,9 @@
     '/etc/anaconda.repos.d'
 ]
 
+# Location of the repository directory in Dracut (may not container repository)
+DRACUT_REPO_DIR = "/run/install/repo"
```

Comment:
Please, move the constant to `pyanaconda.core.constants`. It is used outside of the module.

Reply:
I was already thinking about that.